no-computer.md

```diff
@@ -58,12 +66,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Not really a project, doesn't apply.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Not really a project, doesn't apply.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+Not really a project, doesn't apply.
```

Comment:
Let's say you wanted to implement this as a project. Not necessarily for CS111, but if you had to design a stringed instrument or teach someone how to make music with one, what would you do? Is there anything you would prioritize or change from the way that you learned?